README.md

```diff
@@ -251,15 +228,27 @@ For launch `foggerid` can be obtained from the miner's account:
 
 Choose `nodeid` for for future identification of this machine. It should be unique alphanumeric value without spaces.
 
-Fog Agent will collect the information about your node and register it in your account
+Starting with version 0.7.0, FR Node is part of Fog Agent. Now Faceter FOG Client is a single component. 
```

Comment:
тут появляется FR Node, хотя раньше в описании его не было, стоит написать что-то вроде:
Since version 0.7.0 Fog Agent as part of the Faceter Fog Client starting to be able to download and install all required parts of the Client by it own.